.github/workflows/CI.yml

```diff
@@ -3,14 +3,14 @@ name: CI
 on:
   push:
     branches:
-      - master
+      - main
   pull_request:
   # Allow running manually the CI workflow.
   workflow_dispatch:
 
 # This allows a subsequently queued workflow run to interrupt previous runs.
 concurrency:
-  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  group: '${{ github.workflow }} @ ${{ github.ref }}'
```

Comment:
I think we should do that more. It is taken from homebrew-core repo and the idea is basically to avoid running many times the same job, the github ref being sufficient to know that we don't run this job another way